t/10-jobs.t

```diff
@@ -81,6 +83,10 @@ subtest 'latest jobs' => sub {
     # These are the later clones, they should appear
     ok(grep(/^99963$/, @ids), 'cloned jobs appear as latest job');
     ok(grep(/^99946$/, @ids), 'cloned jobs appear as latest job (2nd)');
+
+    # Jobs can optionally be filtered via "until"
+    @latest = $jobs->latest_jobs(time2str('%Y-%m-%d %H:%M:%S', time - 5 * ONE_MINUTE, 'UTC'));
```

Comment:
I would think this can't work with the broken Jobs.pm code yet it seems to pass?